client/src/common/components/hero/hero.module.scss

```diff
@@ -0,0 +1,62 @@
+@import "colors";
+@import "layout";
+
+.heroWrapper {
+  position: relative;
+  width: 100%;
+  height: 42.6875rem;
+  background-image: url("../../../assets/icons/svg/hero-background.svg");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
+  @include respond-below(sm) {
```

Comment:
Not sure if this matters much here, but wouldn't it make more sense to write mobile styles first and override on bigger screens?